Simplenote/Resources/release_notes.txt

```diff
@@ -1,5 +1 @@
-- Fixed a bug that prevented adding collaborators with email addresses that contain new TLDs
-- Swiping a Note in the List now reveals the Copy Internal Link Action
-- Long Pressing over an Internal Link now pushes the target Note
-- iOS 12 is now the minimum supported OS
-- Password AutoFill is now the preferred way to manage your credentials
+- Added a link to help in Settings and About
```

Comment:
A few more PRs went into this release, but they were all internal code changes. 

I was tempted to added something like "We overhauled our internal dependencies" but it didn't seem too interesting for the non-technical users.